### DISCUSSION TITLE
[Q]  Can i write data into current XLSM file

### DISCUSSION THREAD

**Discussion by ersaurabh101** [exceljs/exceljs#2429] [Q&A]
I have a gvernment GST xls file which has macros,  its extension is xlsm. My question is can xlsm file be read and write using excel js ?

**Comment by LuthfulahiO** [comment]
@ersaurabh101 were you able to figure this out? I am currently having the same issue.

**Comment by sunhao12** [comment]
@LuthfulahiO  were you able to figure this out? I am currently having the same issue.

**Comment by clecherbauer** [comment]
@sunhao12  were you able to figure this out? I am currently having the same issue.

**Comment by jackytank** [comment]
I have the same needs, but I just want to read a .xlsx file -> modify it sheet a bit -> then export it as .xlsm (keeping original format/style/formulae)
